### PR TITLE
Use atom public title as targeting title

### PR DESCRIPTION
**Pull request by annebyrne**
cc @tomrf1 @regiskuckaertz

### CODE REVIEW

File: public/js/components/FormFields/FormFieldTextInput.js [hunk 1]
```diff
@@ -20,10 +20,18 @@ export default class FormFieldTextInput extends React.Component {
   }
 
   render() {
+
     return (
         <div className={this.props.formRowClass || "form__row"}>
           {this.props.fieldLabel ? <label htmlFor={this.props.fieldName} className="form__label">{this.props.fieldLabel}</label> : false}
-          <input type="text" className={"form__field " + (this.props.fieldErrors && this.props.fieldErrors.length ? "form__field--error" : "")}  id={this.props.fieldName} placeholder={this.props.fieldPlaceholder || ''} onChange={this.onUpdate}  value={this.props.fieldValue || ""}/>
+          <input
```
Comment (tomrf1) [inline public/js/components/FormFields/FormFieldTextInput.js]: has anything changed here, or is it just formatting?

Comment (annebyrne) [reply public/js/components/FormFields/FormFieldTextInput.js]: Just formatting! Was difficult to read.

**Comment by tomrf1** [comment]
👍

**Comment by annebyrne** [comment]
@shaundillon are you happy with this being merged?

**Pull request merged**